README.md

```diff
@@ -131,6 +131,47 @@ cpu, err := ghw.CPU(ghw.WithSnapshot(ghw.SnapshotOptions{
 }))
 ```
 
+### Creating snapshots
+
+You can create ghw snapshots in two ways.
+You can just consume the `ghw-snapshot` tool, or you can create them programmatically
+from your golang code. We explore now the latter case.
+
+Snapshotting takes two phases:
+1. clone the relevant pseudofiles/pseudodirectories into a temporary tree
+   This tree is usually deleted once the packing is successful.
+2. pack the cloned tree into a tar.gz
+
+```go
```

Comment:
maybe put the import statement above here?

Reply:
right, done.